README.md

```diff
@@ -299,3 +299,15 @@ spec:
           userDataSecret:
             name: windows-user-data
 ```
+### Windows nodes k8s component upgrade
+
+When a new version of WMCO is released that is compatible with the current cluster version, an update can take place.
+For a non-disruptive upgrade, WMCO terminates the Windows VMs configured by previous version of WMCO and recreates them 
+using the current version. This is done by deleting the Machine object that results in the drain and deletion of the 
+Windows node. To facilitate an upgrade, WMCO adds a version annotation to all the configured nodes. During an upgrade, 
+a mismatch in version annotation will result in deletion and recreation of Windows VM. WMCO takes into consideration the
+`minHealthy`(default=1) count while deleting the nodes. So, at any time during upgrade, the cluster will have
+`minHealthy` Windows machines per MachineSet.
+
+WMCO is not responsible for Windows Operating System updates. The cluster administrator provides the Window image while
+creating the VMs and hence, the cluster administrator is responsible for providing an updated image
```

Comment:
`The cluster administrator can provide an updated image by changing the `image` in the MachineSet...`